cmd/ferretdb/main.go

```diff
@@ -62,6 +64,7 @@ var (
 	handlerF = flag.String("handler", "pg", "<set in initFlags()>")
 
 	testConnTimeoutF = flag.Duration("test-conn-timeout", 0, "test: set connection timeout")
+	logLevel         = flag.String("log-level", "info", "set backend handler (debug/info/warn/error)")
```

Comment:
Let's set both the default value and usage text in `initFlags` function and use zap's constants instead of our own hard-coded values